<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrumDisplay</class>
 <widget class="QFrame" name="SpectrumDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>1</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="MinFreq">
       <property name="suffix">
        <string> Hz</string>
       </property>
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>1300</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="Smooth">
       <property name="text">
        <string>Smooth</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="MaxFreq">
       <property name="suffix">
        <string> Hz</string>
       </property>
       <property name="minimum">
        <number>1400</number>
       </property>
       <property name="maximum">
        <number>2500</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>2500</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QSplitter" name="spectrumSplitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="opaqueResize">
      <bool>true</bool>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="SpectrumWindow" name="spectrumWindow">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>80</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
     </widget>
     <widget class="WaterfallWindow" name="waterfallWindow">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpectrumWindow</class>
   <extends>QFrame</extends>
   <header>spectrumwindow.h</header>
   <container>1</container>
   <slots>
    <signal>frequencyChanged(int)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>WaterfallWindow</class>
   <extends>QFrame</extends>
   <header>waterfallwindow.h</header>
   <container>1</container>
   <slots>
    <signal>frequencyChanged(int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>spectrumWindow</sender>
   <signal>frequencyChanged(int)</signal>
   <receiver>SpectrumDisplay</receiver>
   <slot>setnewFrequency(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>waterfallWindow</sender>
   <signal>frequencyChanged(int)</signal>
   <receiver>SpectrumDisplay</receiver>
   <slot>setnewFrequency(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setnewFrequency(int)</slot>
 </slots>
</ui>
